psci/PSCi.hs

```diff
@@ -59,6 +59,35 @@ import Parser (parseCommand)
 import Completion (completion)
 import Types
 
+-- | The name of the PSCI support module
+supportModuleName :: P.ModuleName
+supportModuleName = P.ModuleName [P.ProperName "$PSCI", P.ProperName "Support"]
+
+-- | Support module, contains code to evaluate terms
+supportModule :: P.Module
+supportModule = 
+  case P.parseModulesFromFiles id [("", code)] of
+    Right [(_, P.Module cs _ ds exps)] -> P.Module cs supportModuleName ds exps
+    _ -> error "Support module could not be parsed"
+  where
+  code :: String
+  code = unlines
```

Comment:
:) Yeah, triple quotes would be nice. The other reason I did it this way was that I thought it might be nice to have a flag to remove the `Eff` dependency, in which case I'd end up gluing lines together one day, but now we have a pretty heavy `Eff` dependency because of the type of `eval`.